site/src/pages/WorkspacesPage/WorkspacesPage.tsx

```diff
@@ -40,12 +39,12 @@ const WorkspacesPage: FC = () => {
   const searchParamsResult = useSafeSearchParams();
   const pagination = usePagination({ searchParamsResult });
 
-  const organizationId = useOrganizationId();
-  const templatesQuery = useQuery(templates(organizationId, false));
+  const { orgId } = useAuthenticated();
```

Comment:
Easier to miss, but I think the call here and the one on line 158 can be combined